docs/modules/ROOT/pages/app_metrics_guide.adoc

```diff
@@ -0,0 +1,83 @@
+= Analyzing App usage using the metrics service
```

Comment:
All the docs pages can be deleted. We can keep the directories and boilerplate for non-user facing docs.